src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-sequences */
+/* eslint-disable prettier/prettier */
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    (this.name = name), (this.health = health), Animal.alive.push(this);
```

Comment:
Using commas to separate statements in the constructor is unconventional and can reduce readability. Consider using semicolons instead.